src/driver.ts

```diff
@@ -51,23 +66,35 @@ export interface Device {
  * or `null` if a matching device cannot be found connected to your computer.
  */
 export const findDevice = (): Device | null => {
-  const matchingDevice = HID.devices().find(
-    (device) =>
-      device.vendorId === VENDOR_ID &&
-      PRODUCT_IDS.includes(device.productId) &&
-      device.usagePage === USAGE_PAGE,
-  );
+  const matchingDevice = HID.devices().find((device) => isLitraDevice(device));
```

Comment:
Nit: Getting the terse benefits of functional programming 😂 

```suggestion
  const matchingDevice = HID.devices().find(isLitraDevice);
```